app/ide-desktop/lib/dashboard/src/components/AriaComponents/Tooltip/Tooltip.tsx

```diff
@@ -75,6 +75,7 @@ export function Tooltip(props: TooltipProps) {
       className={aria.composeRenderProps(className, (classNames, values) =>
         TOOLTIP_STYLES({ className: classNames, ...values })
       )}
+      data-enso-ignore-click-outside
```

Comment:
avoid using app name in markup

Reply:
yeah, agree, but I want to scope the attribute from potential collision, any alternatives?

Reply:
i think it should be fine to scope it to e.g. `.enso-dashboard [data-ignore-click-outside]`

Reply:
the idea being, we control what is inside our app so:
a) if any collision happens then we already know about it
b) it's guaranteed to not be affected by any elements outside the app.

Reply:
Yeah, makes sense. It's not perfect though(We do need to keep in mind to update the selectors if we change them in `index.html`, but I think this should happen rarely